src/jupyter_contrib_nbextensions/nbextensions/scroll_down/readme.md

```diff
@@ -0,0 +1,3 @@
+# Description
```

Comment:
Sorry, I should have caught this earlier, but the main heading in the readme should be the extension name, so that it appears correctly in the built docs at http://jupyter-contrib-nbextensions.readthedocs.io/en/latest/nbextensions.html